Rakefile

```diff
@@ -68,64 +68,12 @@ end
 namespace :buildpack do
   desc "prepares the next version of the buildpack for release"
   task :prepare do
-    deploy = DeployCheck.new(github: "heroku/heroku-buildpack-ruby")
-    unreleased_changelogs = Pathname(__dir__).join("changelogs/unreleased").glob("*.md")
-    if unreleased_changelogs.empty?
-      puts "No devcenter changelog entries on disk in changelogs/unreleased"
-    else
-      next_changelog_dir = Pathname(__dir__).join("changelogs").join(deploy.next_version.to_s)
-
-      next_changelog_dir.mkpath
-
-      unreleased_changelogs.each do |source|
-        dest = next_changelog_dir.join(source.basename)
-        puts "Moving #{source} to #{dest}"
-        FileUtils.mv(source, dest)
-      end
-    end
-
-    changelog_md = Pathname(__dir__).join("CHANGELOG.md")
-    contents = changelog_md.read
-    version_string = "## #{deploy.next_version}"
-    if contents.include?(version_string)
-      puts "Found an entry in CHANGELOG.md for #{version_string}"
-    else
-      new_section = "## Main (unreleased)\n\n#{version_string} (#{Time.now.strftime("%Y/%m/%d")})"
-
-      puts "Writing to CHANGELOG.md:\n\n#{new_section}"
-
-      changelog_md.write(contents.gsub("## Main (unreleased)", new_section))
-    end
-
-    version_rb = Pathname(__dir__).join("lib/language_pack/version.rb")
-    puts "Updating version.rb"
-    contents =  version_rb.read.gsub(/BUILDPACK_VERSION = .*$/, %Q{BUILDPACK_VERSION = "#{deploy.next_version.to_s}"})
-    version_rb.write(contents)
+    puts("Use https://github.com/heroku/heroku-buildpack-ruby/actions/workflows/prepare-release.yml")
   end
 
   desc "releases the next version of the buildpack"
   task :release do
-    deploy = DeployCheck.new(github: "heroku/heroku-buildpack-ruby")
-    puts "Attempting to deploy #{deploy.next_version}, overwrite with RELEASE_VERSION env var"
-    deploy.check!
-
-    if deploy.push_tag?
-      sh("git tag -f #{deploy.next_version}") do |out, status|
-        raise "Could not `git tag -f #{deploy.next_version}`: #{out}" unless status.success?
-      end
-      sh("git push --tags") do |out, status|
-        raise "Could not `git push --tags`: #{out}" unless status.success?
-      end
-    end
-
-    command = "heroku buildpacks:publish heroku/ruby #{deploy.next_version}"
-    puts "Releasing to heroku: `#{command}`"
-    exec(command)
-  end
-  desc "stage a tarball of the buildpack, this runs on github actions to deploy CNB"
-  task :tarball do
-    tarballer = Tarballer.new(name: "heroku-buildpack-ruby", directory: __dir__)
-    tarballer.call
+    puts("Use https://github.com/heroku/heroku-buildpack-ruby/actions/workflows/prepare-release.yml")
```

Comment:
Both the `release` and `tarball` tasks have been deleted here, but the `release` task is presumably still used, since it's what publishes to the buildpack registry? (The GitHub Actions workflow only prepares the release)